src/utils/payloadStringify.js

```diff
@@ -258,16 +268,34 @@ const invalidMaskingRegexWarning = runOneTimeWrapper((e) => {
 });
 
 const shallowMaskByRegex = (payload, regexes) => {
+  const byPassMaskingKeys = [
```

Comment:
I would move this list to [this file](https://github.com/lumigo-io/lumigo-node/blob/13cf15fc85978bb8e6b2387c2fc4bb1d47c8ca7c/src/utils.ts#L23), and also use the constants of the env-var names instead of the names themselves - for consistency.
This will make a higher chance of us remembering to update that list once (and if) a new env-var is added

Reply:
Done